oauth2_http/java/com/google/auth/oauth2/S2A.java

```diff
@@ -81,14 +81,9 @@ public final class S2A {
     this.config = getS2AConfigFromMDS();
   }
 
-  /** @return the mTLS S2A Address from the mTLS config. */
-  public String getMtlsS2AAddress() {
-    return config.getMtlsAddress();
-  }
-
-  /** @return the plaintext S2A Address from the mTLS config. */
-  public String getPlaintextS2AAddress() {
-    return config.getPlaintextAddress();
+  /** @return the cached S2AConfig. */
+  public S2AConfig getConfigFromMDS() {
+    return config;
   }
 
   public static Builder newBuilder() {
```

Comment:
Also, if you want, you can include a static `create()` helper that creates a default instance (i.e. `SecureSessionAgent.create()`).

Reply:
Done in https://github.com/googleapis/google-auth-library-java/pull/1573/commits/7ce029110e5752ae469e46174c97e30d65caa29c